java/.settings/JMRI PanelPro.launch

```diff
@@ -20,5 +20,5 @@
 <listEntry value="org.eclipse.debug.ui.launchGroup.run"/>
 </listAttribute>
 <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_START_ON_FIRST_THREAD" value="true"/>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${project_loc:jmri}"/>
+<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:/JMRI}"/>
```

Comment:
Is this change desired? Sometimes the IDE makes changes on its own that is not desired to get into JMRI master.